### ISSUE TITLE
Change the unit of Free RAM to GB

### ISSUE THREAD

**Issue by Hellis81** [GamerClassN7/HA_Desktop_Companion#46]
I doubt anyone needs the value in KB. 
If we make it GB with one decimal then we still get `0.5` for example which should be fine as a value I believe.
Or possibly two decimals.

**Comment by GamerClassN7** [comment]
you can add

```yaml
accuracy_decimals: 2
filters:
 - multiply: 0.5
```

![image](https://user-images.githubusercontent.com/22167469/190085574-74fa355b-5e83-4326-b3bd-03193d35c2fa.png)

if you default one correct values i can use them as default one also i would stick to (MB) not GB for those systems like mi intel computer stick. so default value is still usefull

**Comment by Hellis81** [comment]
Huh...

```
  - platform: wmic
    wmic_path: Win32_OperatingSystem
    wmic_selector: FreePhysicalMemory
    wmic_namespace: \\root\CIMV2
    name: Free Ram
    unique_id: free_ram
    unit_of_measurement: "GB"
    icon: "mdi:clock"
    entity_category: "diagnostic"
    accuracy_decimals: 2
    filters:
     - divide: 1000000
```

![image](https://user-images.githubusercontent.com/61010215/190089218-9ef24307-6eb5-42a2-a501-b16588883078.png)

**Comment by GamerClassN7** [comment]
hmm i think you need to reregister the sensors since HA is geting value od (unit) only on registration

**Comment by GamerClassN7** [comment]
Also i think conversion is not  1000000  mor of a 1 048 576 since memori values are steped in 1024 not in 1000

**Comment by Hellis81** [comment]
1048576 does not work.  It makes it crash.

But I deleted the device from HA and restarted. 

![image](https://user-images.githubusercontent.com/61010215/190110420-53b5f439-882d-4f88-9767-eadd8222f833.png)

Are you sure "kilobytes" is not hard coded?

**Comment by GamerClassN7** [comment]
that exception is issue with ssl certificate of your server or nabucasa nothing to do with number i will try to resolve that 

no units are not hardcoded
![image](https://user-images.githubusercontent.com/22167469/190138472-c2792dbc-487e-4843-8405-09d425614eb4.png)

also lastline in log before crash would be apriciated !

**Comment by GamerClassN7** [comment]
also from my tests gass is holding units opn his site 
![image](https://user-images.githubusercontent.com/22167469/190141132-fb552487-4439-4695-ba3f-a0842f716180.png)

**Comment by GamerClassN7** [comment]
if you register nev senzor it is ok

**Comment by GamerClassN7** [comment]
![image](https://user-images.githubusercontent.com/22167469/190142247-29028c84-337a-49f4-a31d-6b4a26de1d34.png)
result of complet cleanup of Device

**Comment by GamerClassN7** [comment]
https://community.home-assistant.io/t/why-does-the-unit-of-measure-do-not-change/75163/14

**Comment by Hellis81** [comment]
The error only happened when I had that number there, I switched back to 1000000 and it worked, then switched to 108576 and it crashed again. 
Since I didn't want to scroll through the full log I figured I could just empty it and recreate the issue again. And this time it worked. 🤷‍♂️ 

OK so either purge the history or wait it out.
I'll wait. It doesn't really matter. 
The idea is mainly to stop the constant writing to HA. Having a smaller number means rounding makes it the same for a longer time.
